### PR TITLE
Create SortSentenceStringTest.java

### PR DESCRIPTION
**Pull request by adjamalbek**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

### CODE REVIEW

**Comment by adjamalbek** [comment]
Added Junit Tests. Kindly review it @siriak @yanglbme @debasishbsws

File: DIRECTORY.md [hunk 1]
```diff
@@ -468,22 +478,37 @@
             * zigZagPattern
               * [zigZagPattern](https://github.com/TheAlgorithms/Java/blob/master/src/main/java/com/thealgorithms/strings/zigZagPattern/zigZagPattern.java)
   * test
```
Comment (siriak) [inline DIRECTORY.md:480]:
```suggestion
  * test
    * java
      * com
        * thealgorithms
```

**Review by debasishbsws — CHANGES_REQUESTED** [review]
File name doesn't follow java convention of using pascal case as class names.

**Review by siriak — CHANGES_REQUESTED** [review]
Please make the same changes as in your different PR

**Review by adjamalbek — COMMENTED** [review]
change the class name

**Review by adjamalbek — COMMENTED** [review]
reviewed

**Review by adjamalbek — COMMENTED** [review]
updated

**Comment by debasishbsws** [comment]
Ok, as I can see you have made a PR for the algorithm and made another PR for the test cases. It would be good to put those in a single PR. Because after seeing this PR we _can't_ find the algorithm present, and on the algorithm's PR we are asking to add some JUnit tests.

**Comment by adjamalbek** [comment]
> Ok, as I can see you have made a PR for the algorithm and made another PR for the test cases. It would be good to put those in a single PR. Because after seeing this PR we _can't_ find the algorithm present, and on the algorithm's PR we are asking to add some JUnit tests.

The reason for this is, I firstly submit the src code, and forgot about the test cases, only after reading comment and have added test cases for all the problems and renamed all the classes by following rules, moreover it's my first experience in GIT, please don't judge strictly, next time I will be more careful and submit everything in a single PR. Thank you

**Comment by siriak** [comment]
You can just merge patch-8 branch into patch-4 and the changes will be reflected in the corresponding PR. We do not add algorithms without/separately from tests

**Pull request closed without merging**